python/cudf/cudf/core/_compat.py

```diff
@@ -3,12 +3,10 @@
 import pandas as pd
 from packaging import version
 
+PANDAS_CURRENT_SUPPORTED_VERSION = version.parse("2.2.1")
 PANDAS_VERSION = version.parse(pd.__version__)
-PANDAS_EQ_200 = PANDAS_VERSION == version.parse("2.0.0")
-PANDAS_GE_200 = PANDAS_VERSION >= version.parse("2.0.0")
-PANDAS_GE_201 = PANDAS_VERSION >= version.parse("2.0.1")
+
+
 PANDAS_GE_210 = PANDAS_VERSION >= version.parse("2.1.0")
-PANDAS_GE_214 = PANDAS_VERSION >= version.parse("2.1.4")
-PANDAS_LT_203 = PANDAS_VERSION < version.parse("2.0.3")
 PANDAS_GE_220 = PANDAS_VERSION >= version.parse("2.2.0")
```

Comment:
question: Will we still need `PANDAS_GE_210`, `PANDAS_GE_220`, and `PANDAS_LT_300` in the future?

Reply:
Those are currently required for some breaking API changes pandas has done across versions. So there is no way we can get rid of these 3.

Reply:
Ah we might need this in a non-testing context just in case an API changes in a minor release